extensions/sql-database-projects/src/models/project.ts

```diff
@@ -614,18 +688,25 @@ export class Project {
 /**
  * Represents an entry in a project file
  */
-export class ProjectEntry {
+export abstract class ProjectEntry {
+	type: EntryType;
+
+	constructor(type: EntryType) {
+		this.type = type;
+	}
+}
+
+export class FileProjectEntry extends ProjectEntry {
```

Comment:
FileProjectEntry here cover both file and folder - right?

Reply:
Yup